### DISCUSSION TITLE
如何让自定义SVG的大小随着改变

### DISCUSSION THREAD

**Discussion by ghost** [antvis/X6#3754] [Q&A]
stencil加入自定义的svg，如何在graph上随意改变大小，就像xflow的流程图解决方案那样

![image](https://github.com/antvis/X6/assets/37528714/07871662-cb58-4ccc-a76f-896dbdc8edaf)

**Comment by NewByVector** [comment]
可以使用 [transform](http://x6.antv.antgroup.com/tutorial/plugins/transform) 插件。

**Reply by ghost** [reply]
好的，感谢，我尝试一下

**Reply by ghost** [reply]
引入tranform插件之后，还是不行

**Reply by ghost** [reply]
https://github.com/LiuChang0725/experiment-x6

**Reply by ghost** [reply]
是不是因为markup中svg的d是固定了，所以才不会自动改变

**Reply by NewByVector** [reply]
使用 refD，也就是相对的 d，具体可以查看文档。

**Reply by ghost** [reply]
好的，我先看下

**Reply by ghost** [reply]
感谢，refD确实可以，但是又发现一个新的问题，多path的情况，path的d是没错的，但是曲线使用refD之后就变形了

**Reply by ghost** [reply]
![image](https://github.com/antvis/X6/assets/37528714/8eaabdae-7097-4752-b37a-a81dc92d6837)
使用在线svg渲染是可以生成的，不知道createNode生成的时候为啥有问题

**Reply by Yeti-xxx** [reply]
解决了吗，我也遇到一模一样的问题。。。
